Framework/Utils/include/Utils/MakeRootTreeWriterSpec.h

```diff
@@ -146,6 +146,19 @@ class MakeRootTreeWriterSpec
 {
  public:
   using WriterType = RootTreeWriter;
+  enum struct TerminationPolicy {
+    Process,
+    Workflow,
+  };
+  const std::map<std::string, TerminationPolicy> sTerminationPolicyMap = {
```

Comment:
why prefix `s`? The map doesn't seem to be static.

Reply:
ah, thanks, wanted to make it static in the first place, will fix it